### ISSUE TITLE
question of “MOON” and loss function.

### ISSUE THREAD

**Issue by chaipangpang** [camel007/caffe-moon#10]
1. I do not find the mixed objective optimization，this project no relationship with the paper—“MOON”.
2.Why use the loss function —“EuclideanLoss”？It is better use “Sigmoid Cross-Entropy” or “HingeLoss”？

**Comment by klintqinami** [comment]
"Since CelebA has identical source and target distributions, we define the loss layer in (6) to weight all elements equally during backpropagation – which is equivalent to Euclidean loss between
the network output and the 40 binary attribute values."

[Page 9, MOON : A Mixed Objective Optimization Network for the Recognition of Facial Attributes]
